### PR TITLE
Provision admin/public tokens on each run of SkyPortal (if they don't exist)

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by stefanv** [comment]
@acrellin I could use your help on this one!

**Comment by stefanv** [comment]
Perhaps we don't even need two tokens: the admin token can also upload public data?

**Comment by acrellin** [comment]
@stefanv is the public token for users to actually use, or just for populating the DB? if the latter, then yeah we don't need it

**Comment by stefanv** [comment]
@acrellin Now I generate only one token.

File: skyportal/model_util.py [hunk 1]
```diff
@@ -56,6 +88,10 @@ def setup_permissions():
 
 
 def create_token(permissions, created_by_id, name):
+    """
+    permissions --- ACLs
```
Comment (acrellin) [inline skyportal/model_util.py:92]: we can update the function signature to be clearer throughout

**Comment by acrellin** [comment]
@stefanv what does this PR need? Anything I can help with?

**Comment by stefanv** [comment]
@acrellin I gave it a shot; is this what you meant?

**Comment by stefanv** [comment]
@acrellin I think it's good to go; I just wanted to get your eyes on it, in case I'm doing something crazy with the tokens!

File: skyportal/model_util.py [hunk 2]
```diff
@@ -13,37 +13,69 @@
...
+    admin = add_user('admin@cesium-ml.org', roles=['Super admin'])
+    token_name = 'Initial admin token'
+
+    token = (
+        Token.query
+        .filter(Token.created_by == admin)
+        .filter(Token.name == token_name)
+    ).first()
+
+    if not token:
```
Comment (acrellin) [inline skyportal/model_util.py:56]: Minor nitpick, but `if token is None`

File: skyportal/model_util.py [hunk 3]
```diff
@@ -13,37 +13,69 @@
...
-        super_user.roles.append(admin_role)
-    DBSession().add(super_user)
-    DBSession().commit()
+    add_user(username, roles=['Super admin'], auth=True)
+
+
+def provision_token():
+    """Provision an initial administrative token.
+    """
+    admin = add_user('admin@cesium-ml.org', roles=['Super admin'])
```
Comment (acrellin) [inline skyportal/model_util.py:47]: I think "testadmin" is what's used in the new yaml-based seed user

Comment (acrellin) [reply skyportal/model_util.py:47]: Or is this not meant to map onto that?

Comment (stefanv) [reply skyportal/model_util.py:47]: It can be anything, I guess---it just has to be a user with enough permissions.

**Comment by stefanv** [comment]
@acrellin I *think* this change is unrelated; can you confirm?

**Pull request merged**